package.json

```diff
@@ -104,6 +104,7 @@
     "react-window": "^1.8.5",
     "rebass": "^4.0.7",
     "redux-localstorage-simple": "^2.3.1",
+    "sass": "^1.34.0",
```

Comment:
node-sass is recommended by [create-react-app](https://create-react-app.dev/docs/adding-a-sass-stylesheet/) but does not yet support the M1 chip, so I've used upstream sass instead.